src/Accounts/Accounts/ChangeLog.md

```diff
@@ -19,6 +19,7 @@
 -->
 
 ## Upcoming Release
+* Upgraded Azure.Core from 1.21.0 to 1.22.0.
```

Comment:
Could you please explain the reason of upgrading `Azure.Core`?

Reply:
The Artifact SDK is upgraded to 1.0.0-preview.13, the version of Azure.Core dependency needs to be greater than or equal to 1.22.0, so I upgrade it here.